segway/_util.py

```diff
@@ -342,14 +356,13 @@ def iter_chroms_coords(filenames, coords):
 
 def extract_superlabel(label):
     """
-    label is either an integer or a string with a superlabel and
-    sublabel part, in which case only the superlabel part is
-    returned
+    label is either an integer superlabel, a string superlabel, or a string 
+    with a superlabel and sublabel part separated by a period. In this last case,
+    only the superlabel part is returned. Returns a string.
     """
-    if isinstance(label, str):
-        return int(label.split(".")[0])
-    else:
-        return label
+    if isinstance(label, str_) and '.' in label:
```

Comment:
why is there a `_` added to `str` here?

Reply:
What if there's a `.` in a given string label name?

Reply:
`str_` is a the Numpy string class and is distinct from the Python `str` class. Only the Numpy class can be array elements, likely due to how Numpy arrays pre-allocate memory for items and Python strings are inherently extendable. 

The label is being passed directly from a Numpy array so should be a `str_`. However, for safety, I'll allow it to be either `str_` or `str`.

Reply:
Can you address symbols/names having a `.` in them in this case? There might be a need for better logic when subsegs and string names.